examples/water/src/main.rs

```diff
@@ -580,7 +582,7 @@ impl wgpu_example::framework::Example for Example {
             layout: Some(&terrain_pipeline_layout),
             vertex: wgpu::VertexState {
                 module: &terrain_module,
-                entry_point: "vs_main",
+                entry_point: "vs_main",constants: &Default::default(),
```

Comment:
issue: The lack of spacing b/w these fields in this list make is seem like `rustfmt` either hasn't been run, or is struggling, for some reason. Both are 🙅🏻 no good!

Reply:
I was relying on `rustfmt` to take care of the formatting on save but it seems it's not working in that specific code block due to the call to `wgpu::vertex_attr_array!`.